Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +74,29 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        items = OrderedDict()
+        if self.normalization_method == 0:
+            if self.inversion_method == 0:
+                items['Transformation'] = 'None'
+            else:
+                items['Transformation'] = 'Inversion (' + \
+                    self.inversion_options[self.inversion_method][0] + ')'
+        else:
+            if self.inversion_method == 0:
+                items['Transformation'] = ('Normalization {}'.
+                                            format(self.normalization_options
+                                            [self.normalization_method][0])
+                                            .lower())
```

Comment:
This indentation is not ok. 

Something like this would be better imho:
```py
items['Transformation'] = (
    'Normalization {}'.format(
        self.normalization_options[self.normalization_method][0]).lower())
```
But @janezd should give his opinon on that. :smiling_imp: 

If you bring the required variables in the local scope, all of it may become much more concise:
```py
norm, normopt = self.normalization_method, self.normalization_options
inv, invopt = self.inversion_method, self.inversion_options
```